pkg/cmd/pr/checks/http.go

```diff
@@ -0,0 +1,70 @@
+package checks
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cli/cli/api"
+	"github.com/cli/cli/internal/ghrepo"
+)
+
+type checkRunPayload struct {
+	Name        string
+	Status      string
+	Conclusion  string
+	StartedAt   time.Time `json:"started_at"`
+	CompletedAt time.Time `json:"completed_at"`
+	HtmlUrl     string    `json:"html_url"`
```

Comment:
Nit: go community naming conventions would encourage that all acronyms are spelled uppercase (unless it's first one in an un-exported identifier, e.g. `urlTarget`), so this would be `HTMLURL`